packages/tree-widget/src/components/search-bar/SearchBox.scss

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) 2019 Bentley Systems, Incorporated. All rights reserved.
+* Licensed under the MIT License. See LICENSE.md in the project root for license terms.
+*--------------------------------------------------------------------------------------------*/
+@import "~@bentley/ui-core/lib/ui-core/index";
+
+.tree-widget-uisnip-core-searchbox {
```

Comment:
probably shouldnt be using `uisnip-core` in these classnames either

Reply:
Agreed. Removed.